shell/components/SortableTable/index.vue

```diff
@@ -382,9 +381,11 @@ export default {
 
       // setTimeout is needed so that this is pushed further back on the JS computing queue
       // because nextTick isn't enough to capture the DOM update for the manual refresh only scenario
-      setTimeout(() => {
-        this.watcherUpdateLiveAndDelayed(neu, old);
-      }, 500);
+      if (old && !neu) {
+        this.manualRefreshTimer = setTimeout(() => {
+          this.watcherUpdateLiveAndDelayed(neu, old);
+        }, 1000);
```

Comment:
We should be careful about extending this. Is it definitely needed?

Reply:
Sorry, just seen https://github.com/rancher/dashboard/pull/6747#issuecomment-1228369321. The issue is that the live reload columns aren't rendered in time for watcherUpdateLiveAndDelayed to work, is it something to do with hasLiveColumns or pagedRows?

Reply:
Thanks for the tips on why this might be happening. This is not easy to pinpoint (and I still haven't exactly...) but upon further investigation, there are a bunch of functions with setTimeout's on SortableTable (such as `updateLiveColumns`) which might mess up the "update queue" for the whole lot...

Still don't know what else to do here to improve this 🤔